File: hash_table.py
STATUS_CODES = {
    200: "OK",
    301: "Moved Permanently",
    401: "Unauthorized",
    404: "Not Found",
    500: "Internal Server Error",
}


def status_code_meaning(number):
    return STATUS_CODES[number]


print(status_code_meaning(200))

File: hashTable.ts
// A hash table of status codes and their corresponding messages.

// The key is a number, and the pair is a string.
const STATUS_CODES = new Map<number, string>()
STATUS_CODES
	.set(200, "OK")
	.set(201, "Created")
	.set(204, "No Content")
	.set(400, "Bad Request")
	.set(401, "Unauthorized");

    // The type returned from the function will be a string.
    const statusCodeMeaning = (): string => {
        return STATUS_CODES.get(400) || "Invalid status code, please try again";
    }

    console.log(statusCodeMeaning());